ui/design-system-core/src/components/Pill/index.tsx

```diff
@@ -33,7 +33,7 @@ const Pill: React.FC<PillProps> = ({
       hover={false}
       label={label}
       size={size}
-      icon={icon}
+      {...(active && { icon: icon })}
```

Comment:
```suggestion
      icon={ active ? icon : null }
```

Reply:
Thanks Didd. I was using '' and typescript complained but using null works perfectly 👍